src/layouts/shortcodes/cloud-prepare/permissions.md

```diff
@@ -0,0 +1,9 @@
+The following permissions need to be enabled in IAM on the `openshift-machine-api` user for your cluster:
+
+* CreateTags
```

Comment:
Can we add aws CLI bash to configure those permissions?

We are moving from something that it's copy & paste in the cmdline and can even be automated, to something that involves logging into a console and manual-click.

Reply:
There's an issue for that, in the meantime this documentation is accurate.

`prep_for_subm.sh` still exists and can be used but won't be documented for 0.10

Reply:
@mkolesnik IIUC, the issue if for handling the permission issue properly. But until we handle that, is there a better way we can show/suggest other than going to the AWS web console?

Reply:
I suggest we can do it for a follow up PR for the website since itll take a bit to figure out how to do, since the user name is generated and some information in the permissions might be.

Reply:
Opened #509 for this

Reply:
Ack, either a backport of the future fix or #509 sounds good to me.